guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,7 +1,12 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
-When you run a remote job on hosts, for every host, {Project} performs the following actions to find a remote execution {SmartProxy} to use.
+Remote execution can be performed on the hosts that are registered through a {SmartProxy}.
+You must enable *remote_execution_prefer_registered_through_proxy* manually.
```

Comment:
This still needs to be reworded IMHO.

When describing in this concept _what_ REX is, we should not immediately start with "you'll probably what to use this very special feature; here's how to activate it".

Can you please provide some more details on _why_ we recommend enabling `remote_execution_prefer_registered_through_proxy`?

I'd also switch the order and add this below the procedure starting in line 11.

Reply:
I guess we dropped the idea of `recommending`. Instead, we kept that way optional for end-user. I hope I understood correctly from earlier comments. 

For other changes that you suggested, I am Okay with that :+1:

Reply:
Hi @maximiliankolb 

After my further discussion with SME, we are in agreement to provide some more details on why we recommend `enabling remote_execution_prefer_registered_through_proxy`?

I will add more text for the same. :+1:

Reply:
Addressed :+1: